### PR TITLE
Add container parameters to the create modal

### PR DESCRIPTION
**Pull request by Poltergeist**
Add the `Parameters` section to the container settings section.

image

image

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @pierlo-upitup

File: src/js/schemas/service-schema/ContainerSettings.js [hunk 1]
```diff
@@ -58,6 +58,41 @@ let ContainerSettings = {
...
+      duplicable: true,
+      addLabel: 'Add Parameter',
+      getter: function (service) {
+        let container = service.getContainerSettings();
+        if (container && container.docker &&
+          container.docker.parameters
+        ) {
+          let parameters = container.docker.parameters;
+
+          return Object.keys(parameters).reduce(function (memo, key) {
```
Comment (kennyt) [inline src/js/schemas/service-schema/ContainerSettings.js]: sorry for this again, but what do you think about using `map` here?

Comment (Poltergeist) [inline src/js/schemas/service-schema/ContainerSettings.js]: Done!

**Comment by pierlo-upitup** [comment]
Just FTR – this PR didn't correctly introduce the right handling of docker parameters, which is addressed in https://github.com/dcos/dcos-ui/pull/672.

**Pull request merged**